openstack/messaging/v2/queues/requests.go

```diff
@@ -196,3 +196,38 @@ func GetStats(client *gophercloud.ServiceClient, queueName string) (r StatResult
 		OkCodes: []int{200}})
 	return
 }
+
+// ShareOpts specifies share creation parameters.
+type ShareOpts struct {
+	Paths   []string `json:"paths,omitempty"`
```

Comment:
It looks like there are only a limited number of paths:

https://github.com/openstack/zaqar/blob/28f1f0f9370c395023bc5ae01693e5d952b3e1c1/zaqar/transport/wsgi/v2_0/urls.py#L31

We _could_ make a custom type for these, but that feels kind of weird to me for some reason. If you feel a custom type would be beneficial, by all means go for it, but I'm also good with leaving this as `[]string`.

Reply:
I am going to leave this as string. The path would have the dynamic queue name in it. I will however update the unit tests to reflect this being the case.

```/v2/queues/test/subscriptions```